protocol/requests/create_account.go

```diff
@@ -65,12 +78,14 @@ type WalletSecretsConfig struct {
 }
 
 func (c *CreateAccount) Validate() error {
-	return ValidateAccountCreationRequest(*c)
+	return ValidateAccountCreationRequest(*c, false)
 }
 
-func ValidateAccountCreationRequest(c CreateAccount) error {
+func ValidateAccountCreationRequest(c CreateAccount, allowEmptyDisplayName bool) error {
 	// TODO(cammellos): Add proper validation for password/displayname/etc
```

Comment:
Hmmm, also this function should probably be 
```suggestion
func (c *CreateAccount) ValidateAccountCreationRequest(allowEmptyDisplayName bool) error {
	// TODO(cammellos): Add proper validation for password/displayname/etc
```

I know you didn't write this, but this function deals exclusively with the internal state of the struct so makes sense to be a struct function, rather than taking the struct as an argument. ... Unless I'm missing something.

Reply:
Well there is such a function actually 😄 
And it's used as a wrapper around `ValidateAccountCreationRequest`.

https://github.com/status-im/status-go/blob/1886dc336041b41623187be57f43ac0d855615d3/protocol/requests/create_account.go#L69-L71

But I agree, I'll refactor this 👍